core/error.rs

```diff
@@ -226,7 +241,7 @@ impl JsError {
               .ok();
           let eval_origin = eval_origin.map(|s| s.to_rust_string_lossy(scope));
           let is_top_level: Option<v8::Local<v8::Boolean>> =
-            get_property(scope, call_site, "isTopLevel")
+            get_property(scope, call_site, "isToplevel")
```

Comment:
Typo?

Reply:
It's a fix of an old typo, yeah. V8 spells it `isToplevel`.